kolibri/plugins/coach/assets/src/views/ClassLearnersListPage.vue

```diff
@@ -61,9 +61,9 @@
                 />
               </td>
               <td>
-                <span dir="auto">
-                  {{ $tr('lastSyncedTime', { time: mapLastSynctedTimeToLearner(learner.id) }) }}
-                </span>
+                <ElapsedTime
+                  :date="mapLastSyncedTimeToLearner(entry)"
```

Comment:
With fresh eyes, seems like `entry` was maybe pulled from some of my code in `mapLastSyncedTimeToLearner` that you later refactored and now is undefined... Maybe it should be `learner.id`?

Reply:
You mean there might be a bug in the code I refactored but did not test? This seems possible...

Reply:
```suggestion
                  :date="mapLastSyncedTimeToLearner(learner.id)"
```